ValidatorPanel.php

```diff
@@ -858,33 +866,45 @@ public function stopBuffering()
 	/**
 	 * Validate.
 	 */
-	public function validate()
+	public function validate(\Nette\Application\Application $application)
 	{
 		if (!ob_get_level()) {
 			return;
 		}
+		
+		$validationEnabled = TRUE;
+		
+		foreach($this->disableValidationInPresenters as $disableValidationInPresenter){
```

Comment:
Are you sure about this? Personally, I'd check whether `$application->getPresenter()` is in `$this->disableValidationInPresenters` and if it is so, I'd exit early. I believe it would look like this:

```php
if(in_array(get_class($application->getPresenter()), $disableValidationInPresenter, true)){
    return;
}
```

Then we can completely throw away `$validationEnabled` I guess.

Reply:
Yes it´s easier way how to do this. Only one thing was wrong, you have to call ob_end_flush() first before you can return.